waku/v2/node/waku_node.nim

```diff
@@ -392,10 +392,11 @@ proc registerRelayDefaultHandler(node: WakuNode, topic: PubsubTopic) =
     return
 
   proc traceHandler(topic: PubsubTopic, data: seq[byte]) {.async, gcsafe.} =
+    let mh = MultiHash.digest("sha2-256", data).expect("valid hash")
```

Comment:
Why don't we use the std/hash `sha256` function directly? Is there any requirement for using multihash here?

Reply:
Yes, good catch - we should use only `sha256`. We used Multihash before to mimic what we used in the messageIdProvider, but that has also since changed to pure `sha256` to match with go-waku.

Reply:
btw can we use `digest` from `digest*(pubsubTopic: PubsubTopic, msg: WakuMessage): WakuMessageDigest =`? or what is stopping is from using it? `timestamp` is not included but that should be encoded in `meta`? [as per this](https://github.com/waku-org/nwaku/blob/master/waku/v2/protocol/waku_message/message.nim#L32)?